lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -37,8 +37,12 @@ use LedgerSMB::Magic qw( EC_EMPLOYEE HTTP_454 PERL_TIME_EPOCH );
 use HTTP::Status qw( HTTP_OK HTTP_UNAUTHORIZED );
 
 my $logger = Log::Log4perl->get_logger('LedgerSMB::Scripts::setup');
-$LedgerSMB::VERSION =~ /(\d+\.\d+)./;
-my $MINOR_VERSION = $1;
+my $MINOR_VERSION;
+if ( $LedgerSMB::VERSION =~ /(\d+\.\d+)./ ) {
+    $MINOR_VERSION = $1;
+} else {
+        die "Error: Invalid data";
```

Comment:
@sbts actually, the $MINOR_VERSION value is later interpolated into strings to find an upgrade script. I'd say that the only way to find that that no longer works is to have the setup script fail early. It won't hit any users, I assume, but it'll make Travis show we changed something that's not compatible with other modules, when we do.